pkg/workers/push/push.go

```diff
@@ -148,7 +150,20 @@ func pushToAndroid(ctx *jobs.WorkerContext, msg *Message) error {
 		priority = "high"
 	}
 
-	hashedSource := hashSource(msg.Source)
+	var hashedSource []byte
+	if msg.Collapsible {
+		hashedSource = hashSource(msg.Source)
+	} else {
+		hashedSource = hashSource(msg.Source + msg.NotificationID)
+	}
+
+	// notID should be an integer, we take the first 32bits of the hashed source
+	// value.
+	notID := int32(binary.BigEndian.Uint32(hashedSource[:8]))
```

Comment:
Why do yo take 8 bytes (64 bits) from `hashedSource` if you just want 32 bits?

Reply:
4 bytes is enough yes.